wechatpay/README.md

```diff
@@ -0,0 +1,55 @@
+# Stripe Android SDK WeChatPay module
+Module to provide support for `PaymentIntent` with `wechat_pay` as its payment method.
+ 
+# Overview
+* This module will provide a `PaymentAuthenticator` implementation to authenticate a [PaymentIntent](https://stripe.com/docs/api/payment_intents) with [next_action.wechat_pay_redirect_to_android_app](https://stripe.com/docs/api/payment_intents/object#payment_intent_object-next_action-wechat_pay_redirect_to_android_app) by invoking the WeChat Pay SDK and pass the required parameters from that hash.
```

Comment:
```suggestion
* This module will provide a `PaymentAuthenticator` implementation to authenticate a [PaymentIntent](https://stripe.com/docs/api/payment_intents) with [next_action.wechat_pay_redirect_to_android_app](https://stripe.com/docs/api/payment_intents/object#payment_intent_object-next_action-wechat_pay_redirect_to_android_app) by invoking the WeChat Pay SDK and passing the required parameters from that hash.
```
```suggestion
* This module will provides a `PaymentAuthenticator` implementation for authenticating .a [PaymentIntent](https://stripe.com/docs/api/payment_intents).  To do this it invokes the WeChat Pay SDK to retrieve a hash, and sets this value in the [PaymentIntent.next_action.wechat_pay_redirect_to_android_app](https://stripe.com/docs/api/payment_intents/object#payment_intent_object-next_action-wechat_pay_redirect_to_android_app).
```

Reply:
I tried splitting the sentence up to make it a bit clear.  I tried to make it so it first stating that it provides and Authenticator and then, in a second sentence, stating what it does.   

I am having trouble editing the suggestion now.  There is a '.' after the word authenticating that should be removed.